### ISSUE TITLE
[BUG]. BarcoPDS Module Error Code ISEL-e9998

### ISSUE THREAD

**Issue by TreborKutscher** [bitfocus/companion-module-barco-pds#12]
Hey i got a annoying Error Message From Barco PDS:

Every time i retrigger the preview input to the same input it was before. There is the answer:
“Opperation not applicable .... ISEL -e 9998”

Used witch chained wich select input and then take. Prior to manually check the input on prv to adjust and check.

The Module works fine then. Maybe this is how the PDS works internally and discards doubled commands.

**Comment by MarcHagen** [comment]
Hi there!

Could you give me some information about what version companion you're running?
I have not seen this error personally in the latest and greatest builds.

If possible please share your companion config.

**Comment by TreborKutscher** [comment]
Hello Marc,

The version  i run is: v2.1.0 (v2.1.0-47823d0-2493) also after changing to 2.1.0 this message shows the first time.

Previously on 2.0. this error did not show by the same usage.

**Comment by MarcHagen** [comment]
Oké cool, I’ll check what’s happening.

**Comment by dnmeid** [comment]
"ISEL -e 9998" means the PDS can't select the input at that time. There are two probable reasons for that error:
1. The PDS is transitioning at the time. You can't change input selection during transition.
2. The PDS is still busy doing the input selection. When you select an input (even if it is the same) the PDS tries to detect the input signal status and load the suitable file. During that time you can't change the input selection.

Please make sure that there is enough time between the actions if you send multiple commands to the PDS with one button.

**Comment by TreborKutscher** [comment]
Hello mr. mind.

To 1. Yes i can confirm that this happend. i usaly use on absulte delay: 10ms "select input" then on 200ms "take". After i set the the delay higher to "take" at 300ms i could avoid this error.

To 2.  how long is this period? If i recrate the "lang rpc" with each input as an seperate button and a dedicated take button i was also able to trigger this error.
Only using the select input comand. If selecting different inputs for preview in a row and relative quick by hand all is good. But when i press "input5" wait aproximatly one minute and then trigger "Input5" again this error shows off. This occurs with every double select input for preview. but i whill test this again if i get hands on a pds-902 again.

Greetings Paul

**Comment by MarcHagen** [comment]
Yes double selecting the same input for preview triggers this error. Because the preview is already on the selected input.

In a up coming version the error text is better. But this error is intended by the Barco firmware.
You can ignore this "error" message safely.

**Comment by TreborKutscher** [comment]
Thank you for this clarification. 
This what i assumed afer i could manualy trigger this behaviour. But i used to work like this and getting this error messages was new for me in 2.1.0 , i mostly have the log open to see if one of my devices gets offline.
Now i know thath this error is not a risky bussines. Thanks a lot.
